backend/src/question.ts

```diff
@@ -0,0 +1,105 @@
+import { APIGatewayEvent, ProxyResult } from 'aws-lambda';
+import { Question, QuestionOption } from './models';
+import responses from './util/API_Responses';
+
+// GET /api/v1/question
+const get = async (event?: APIGatewayEvent): Promise<ProxyResult> => {
```

Comment:
Another thing is removing the `?` op from the function variables 
event is always going to be set it will never b undefined.